docker-bake.hcl

```diff
@@ -0,0 +1,111 @@
+# VARIABLES
+
+variable "BUILD_TAG" {
+  default = "latest"
+}
+
+variable "SEMVER" {
+  default = "v0.0.0"
+}
+
+variable "GITCOMMIT" {
+  default = "dev"
+}
+
+variable "GITDATE" {
+  default = "0"
+}
+
+# GROUPS
+
+group "default" {
+  targets = ["all"]
+}
+
+group "all" {
+  targets = ["node-group", "disperser-group"]
+}
+
+group "node-group" {
+  targets = ["node", "nodeplugin"]
+}
+
+group "disperser-group" {
+  targets = ["batcher", "disperser", "encoder", "retriever", "churner"]
```

Comment:
Technically retriever and churner are not part of disperser

Reply:
ah good point will leave them as separate individual targets then.

Reply:
[97b4eed](https://github.com/Layr-Labs/eigenda/pull/754/commits/97b4eedce69bfd32aef7b2209d2081a7d14ebc6b)